src/transformState.js

```diff
@@ -6,6 +6,27 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const action of actions) {
+    switch (true) {
+      case action.type === 'addProperties':
```

Comment:
```suggestion
  for (const action of actions) {
    switch (action.type) {
      case 'addProperties':
```
I think it will be better:
the expression inside switch (action.type) is compared with the condition in case. And if true - code will be executed, if false - break will be executed